src/main/markers/waveform.segments.js

```diff
@@ -25,9 +26,9 @@ define([
       return view;
     });
 
-    var createSegmentWaveform = function (segmentId, startTime, endTime, editable, color, labelText) {
+    var createSegmentWaveform = function (id, startTime, endTime, editable, color, labelText) {
```

Comment:
I recommend keeping parameters in the same order between this and `createSegment`, and `addSegment` in main.js.